package.json

```diff
@@ -16,14 +16,16 @@
     "@expo/metro-runtime": "~3.1.3",
     "@expo/vector-icons": "^14.0.0",
     "@react-native-community/datetimepicker": "^7.6.4",
+    "@react-native-google-signin/google-signin": "^11.0.1",
     "@react-navigation/bottom-tabs": "6.5.20",
     "@react-navigation/native": "6.1.17",
     "@react-navigation/stack": "6.3.29",
-    "eas": "^0.1.0",
     "@testing-library/react-native": "^12.4.0",
+    "eas": "^0.1.0",
```

Comment:
why did you install eas? I think it is a mistake

Reply:
You're right thanks